examples/demo-stf/src/sov-cli/main.rs

```diff
@@ -0,0 +1,200 @@
+use anyhow::Context;
+use borsh::BorshSerialize;
+use clap::{Args, Parser, Subcommand};
+use sov_default_stf::RawTx;
+use sov_modules_api::transaction::Transaction;
+use std::fs;
+use std::fs::File;
+use std::io::Write;
+use std::path::{Path, PathBuf};
+
+use demo_stf::runtime::cmd_parser;
+
+use sov_modules_api::{
+    default_context::DefaultContext, default_signature::private_key::DefaultPrivateKey,
+    AddressBech32, PublicKey, Spec,
+};
+
+type C = DefaultContext;
+type Address = <C as Spec>::Address;
+
+#[derive(Parser)]
+#[command(author, version, about, long_about = None)]
+struct Cli {
+    #[command(subcommand)]
+    command: Commands,
+}
+
+#[derive(Subcommand)]
+enum Commands {
+    SerializeCall {
+        sender_priv_key_path: String,
+        module_name: String,
+        call_data_path: String,
+        nonce: u64,
+    },
+    Util(UtilArgs),
+}
+
+#[derive(Args)]
+struct UtilArgs {
+    #[command(subcommand)]
+    command: UtilCommands,
+}
+
+#[derive(Subcommand)]
+enum UtilCommands {
+    DeriveTokenAddress {
```

Comment:
The same for `/// comments` here. Let's add some more info here and check if they are available with `--help`